main/User_config.h

```diff
@@ -454,28 +454,42 @@ CRGB leds[FASTLED_IND_NUM_LEDS];
 #  ifndef RGB_LED_POWER
 #    define RGB_LED_POWER -1 // If the RGB Led is linked to GPIO pin for power define it here
 #  endif
+#  ifndef FASTLED_BRIGHTNESS
+#    define FASTLED_BRIGHTNESS 20 // Set Default RGB brightness to approx 10% (0-255 scale)
+#  endif
+// Allow to set LED used (for example thingpulse gateway has 4 we use them independently)
+#  ifndef FASTLED_INFO_LED
+#    define FASTLED_INFO_LED 0 // First Led
+#  endif
+#  ifndef FASTLED_SEND_RECEIVCE_LED
+#    define FASTLED_SEND_RECEIVCE_LED 0 // First Led
+#  endif
+#  ifndef FASTLED_ERROR_LED
+#    define FASTLED_ERROR_LED 0 // First Led
+#  endif
+
 #  define SetupIndicators()                                                               \
     pinMode(RGB_LED_POWER, OUTPUT);                                                       \
     digitalWrite(RGB_LED_POWER, HIGH);                                                    \
     FastLED.addLeds<FASTLED_IND_TYPE, FASTLED_IND_DATA_GPIO>(leds, FASTLED_IND_NUM_LEDS); \
-    FastLED.setBrightness(20);
-#  define ErrorIndicatorON() \
-    leds[0] = CRGB::Orange;  \
+    FastLED.setBrightness(FASTLED_BRIGHTNESS);
+#  define ErrorIndicatorON()             \
+    leds[FASTLED_ERROR_LED] = CRGB::Red; \
```

Comment:
Could you set it to Orange please, I have found the Red to be a bit violent for errors

Reply:
Done